saleor/graphql/discount/mutations.py

```diff
@@ -382,4 +403,83 @@ def perform_mutation(cls, _root, info, **data):
             info, data.get("id"), only_type=Sale, field="sale_id"
         )
         cls.remove_catalogues_from_node(sale, data.get("input"))
-        return SaleRemoveCatalogues(sale=sale)
+        return SaleRemoveCatalogues(sale=ChannelContext(node=sale, channel_slug=None))
+
+
+class SaleChannelListingAddInput(graphene.InputObjectType):
+    channel_id = graphene.ID(required=True, description="ID of a channel.")
+    discount_value = PositiveDecimal(description="The value of the discount.")
```

Comment:
I think this vale should be required.
Whats means if the user wants to create/update `SaleChannelListing` in the channel without value?